ui/pages/onboarding-flow/onboarding-app-header/onboarding-app-header.test.js

```diff
@@ -0,0 +1,41 @@
+import { fireEvent } from '@testing-library/react';
+import React from 'react';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { renderWithProvider } from '../../../../test/lib/render-helpers';
+import OnboardingAppHeader from './onboarding-app-header';
+
+const mockUpdateCurrentLocale = jest.fn();
+
+jest.mock('../../../../app/_locales/index.json', () => {
+  return [{ code: 'en', name: 'English' }];
+});
+
+jest.mock('../../../store/actions.ts', () => ({
+  updateCurrentLocale: () => mockUpdateCurrentLocale,
+}));
+
+describe('OnboardingAppHeader', () => {
+  const mockState = {
+    localeMessages: {
+      currentLocale: 'en',
+    },
+  };
+
+  const store = configureMockStore([thunk])(mockState);
+
+  it('should match snapshot', () => {
+    const { container } = renderWithProvider(<OnboardingAppHeader />, store);
+
+    expect(container).toMatchSnapshot();
```

Comment:
Can you try to expand the dropdown and check the snapshot of all `options`?

Reply:
I don't understand. The snapshot already have the `options` presented under the `select`. 
[onboarding-app-header.snapshot#L206-L214](https://github.com/MetaMask/metamask-extension/pull/18254/files#diff-2dd1dde7050384fc07112361ea2b9105bbd0573b4a9baa39114b916528994939R206-R214).

Reply:
My bad !